### DISCUSSION TITLE
Salesforce CLI update needed

### DISCUSSION THREAD

**Discussion by manas34** [microsoft/winget-pkgs#132027] [Ideas]
Salesforce CLI which is currently added in Winget is old and doesn't work anymore.
here is the new latest URL which should be added instead, if you see they have replaced 'sfdx' to 'sf' now.
https://developer.salesforce.com/media/salesforce-cli/sf/channels/stable/sf-x64.exe 

Source: https://developer.salesforce.com/tools/salesforcecli

**Comment by Trenly** [comment]
Please open an issue instead of a discussion in the future, it will be seen faster
